csv_detective/detect_fields/FR/geo/latitude_l93/__init__.py

```diff
@@ -1,13 +1,20 @@
+from frformat import LatitudeL93
 from csv_detective.detect_fields.other.float import _is as is_float
+from csv_detective.detect_fields.other.float import float_casting
+
 
 PROPORTION = 0.9
 
 
 def _is(val):
-    '''Renvoie True si val peut etre une latitude en Lambert 93'''
     try:
-        is_float(val) and float(val) >= 6037008 and float(val) <= 7230728
-    except ValueError:
+        if type(val) is float or type(val) is int:
```

Comment:
Maybe `isinstance(val, (float, int))` ?

Reply:
Yes, it's a good idea to replace the method "type()", thank you.